app/controllers/api/v1/rental_assistances_controller.rb

```diff
@@ -34,7 +34,8 @@ def update
         )
 
         if response
-          render json: true
+          updated_rental_assistances = soql_rental_assistance_service.application_rental_assistances(params[:application_id])
```

Comment:
this also looks like changes from your other PR

Reply:
removed

Reply:
wondering which state is the right state to be in. Do you remember which it was supposed to be from the end of your last PR?